### PR TITLE
tools: wasptool: can now check battery level

### PR DESCRIPTION
**Pull request by thefenriswolf**
This adds a battery check for wasptool, which I found quite handy at times.
Let me know if I need to change anything for this pr to be accepted.

Signed-off-by: thefenriswolf <stefan.rohrbacher97@gmail.com>

### CODE REVIEW

File: wasp/fonts/__init__.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # SPDX-License-Identifier: LGPL-3.0-or-later
 # Copyright (C) 2020 Daniel Thompson
 
+import fonts.sans18 as sans18
```
Comment (Kozova1) [inline wasp/fonts/__init__.py:4]: I don't think this is related to the PR, is it?

File: wasp/fonts/sans18.py [hunk 2]
```diff
@@ -0,0 +1,264 @@
+# Code generated by font_to_py.py.
```
Comment (Kozova1) [inline wasp/fonts/sans18.py:1]: Wouldn't this be better in a separate PR?

File: wasp/requirements.txt [hunk 3]
```diff
@@ -1,6 +1,7 @@
 cbor
 click
 cryptography
+freetype-py
```
Comment (Kozova1) [inline wasp/requirements.txt:4]: As I said above ^^

**Review by Kozova1 — CHANGES_REQUESTED** [review]

File: wasp/fonts/sans18.py (hunk 2 above)
Comment (thefenriswolf) [reply wasp/fonts/sans18.py:1]: You are absolutely correct, this was intended to be a separate PR.
I have no idea how I messed this up

Comment (Kozova1) [reply wasp/fonts/sans18.py:1]: If you'd run these commands it should fix it:

```sh
git rebase -i HEAD^^
```
And write "drop" next to the font commit.
After that, create a new branch and run this command:
```sh
git cherry-pick 919f8b86de5704a3749eb565a83b8c0b8d9d1d1d
```
And submit a separate PR for it.

Comment (Kozova1) [reply wasp/fonts/sans18.py:1]: Make sure that your new branch is based on **this** master, not the one you use - otherwise, your font PR will have both the font and the battery level changes.

Generally, it's easier to work with a master branch that's synced to this branch, and each time you want to add a feature branch from it and add your feature there.

Comment (Kozova1) [reply wasp/fonts/sans18.py]: Always glad to help :)

**Comment by daniel-thompson** [comment]
Applied. Thanks.
